extensions/wikia/ContentReview/models/ReviewModel.php

```diff
@@ -130,4 +130,18 @@ public function updateReviewById( $reviewId, $revisionId, $submitUserId ) {
 			throw $e;
 		}
 	}
+
+	public function getContentToReviewFromDatabase() {
+		$db = $this->getDatabaseForRead();
+
+		$content = ( new \WikiaSQL() )
+			->SELECT( '*' )
+			->FROM( 'content_review_status' );
+
+		$content = $content->runLoop( $db, function( &$content, $row ) {
```

Comment:
I'd suggest using a different variable for the result than for the query object.